src/App.js

```diff
@@ -1,27 +1,39 @@
-import logo from './logo.svg';
-import './App.css';
+// import './App.css';
+import Wrapper from './components/Wrapper';
+import Display from './components/Display';
+import Button from './components/Button';
+import ButtonPanel from './components/ButtonPanel';
+
+const btnValues = [
+  ['C', '+-', '%', '/'],
+  [7, 8, 9, 'X'],
+  [4, 5, 6, '-'],
+  [1, 2, 3, '+'],
+  [0, '.', '='],
+];
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] The React.fragment short syntax should be used to wrap the children components for this function, please do so to comply with the project requirements: Has children components wrapped in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))